syntax = "proto3";
package task;




import "capacity_admin_sdk/model/capacity_admin/task.proto";

/*
ListTasks请求
*/
message ListTasksRequest {
    /*
    页码
    */
    int32 page = 1 ;
    /*
    每页大小
    */
    int32 pageSize = 2 ;
    /*
    任务状态
    */
    string status = 3 ;
    /*
    查询起始时间
    */
    string startTime = 4 ;
    /*
    查询结束时间
    */
    string endTime = 5 ;
    /*
    用户名
    */
    string user = 6 ;
}

/*
ListTasks返回
*/
message ListTasksResponse {
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated capacity_admin.CapacityTask list = 4 ;
}

/*
ListTasksApi返回
*/
message ListTasksResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    task.ListTasksResponse data = 4 ;
}

